packages/eds-core-react/src/components/TopBar/TopBar.stories.tsx

```diff
@@ -15,6 +15,9 @@ import {
 export default {
   title: 'Navigation/TopBar',
   component: TopBar,
+  args: {
+    elevation: 'raised',
```

Comment:
Suggest removing this or set it to `none` as thats the default elevation for `Topbar`

Reply:
The example is a sticky topbar though, which is the use case where it should be elevated. Perhaps the main example should not be sticky and move that down as its own story?

Reply:
eh, I just removed it